### PR TITLE
add hub commands

### PR DESCRIPTION
**Pull request by moshe-kabala**
Please review just the third commit [Add export import and publish on project](https://github.com/tensorleap/leap-cli/pull/87/commits/01fc7a331741b633284dd4ec0cc22316ba08b88d)

### CODE REVIEW

File: cmd/projects/export.go [hunk 1]
```diff
@@ -0,0 +1,42 @@
+package projects
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/tensorleap/leap-cli/pkg/entity"
+	"github.com/tensorleap/leap-cli/pkg/project"
+)
+
+func NewExportCmd() *cobra.Command {
```
Comment (asafyehezkel) [inline cmd/projects/export.go:9]: Even though the command is under projects package I think it should renamed to ExportProject

File: cmd/projects/publish.go [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+				}
+
+				err = hub.HandleProjectExistsOnPublish(ctx, hubApi, meta, &projectContext.Meta, override)
+				if err != nil {
+					return err
+				}
+
+				if useSignedUrl {
+					return StreamProjectToHubBySignedUrl(ctx, hubApi, projectContext)
+				} else {
```
Comment (asafyehezkel) [inline cmd/projects/publish.go:57]: remove redundant else

File: cmd/projects/publish.go [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+					return StreamProjectToHub(ctx, hubApi, projectContext)
+				}
+			}
+
+			projectToPublish, err := project.SelectProjectsToPublish(ctx, projectNames, all)
```
Comment (asafyehezkel) [inline cmd/projects/publish.go:62]: `projectsToPublish` if it is many

**Review by asafyehezkel — APPROVED** [review]
LGTM

File: cmd/projects/export.go (hunk 1 above)
Comment (moshe-kabala) [reply cmd/projects/export.go:9]: This is the convention so far

**Review by YuvalFeldman — APPROVED** [review]

**Pull request merged**